testing/test-common.sh

```diff
@@ -51,6 +60,26 @@ function run_tests() {
         run="--recursive ${TEST_ROOT}/smoke ${TEST_ROOT}/integration/${TEST_PLATFORM}"
     fi
 
+    if [ "$smoke" -eq 1 ]; then
+        export SMOKE_NODES=3
+        start_smoke_cluster
+        trap teardown_smoke_cluster EXIT
+    fi
+
+    set +x
+
+    echo
+    echo
+    echo "========================"
+    echo
+    echo
+
     # shellcheck disable=SC2086
     bats --formatter "${BATS_FORMATTER}" $run --timing
+
+    echo
+    echo
+    echo "========================"
+    echo
+    echo
```

Comment:
Generally I like a nice "All tests passed" at the end if they did, much easier than checking anything else.